Transbank/Onepay/Net/Channel.cs

```diff
@@ -6,45 +6,42 @@
 
 namespace Transbank.Onepay.Net
 {
-    public abstract class Channel
+      public abstract class Channel
```

Comment:
You have an extra space here.

Reply:
1. For catchig this mistakes I use git diff  from the cli, but maybe VisualStudio has a built in diff tool.
2. There is a Test Project alongside the SDK, you can run it locally using `dotnet test TransbankTest` or using the test explorer in VS.
3. Travis runs on each PUSH and PR (also Travis is a WIP) you have to look for SemaphoreCI wich also runs on every PUSH and PR. 

This will be son clearer inside the README, sorry for the delay, we are still working on the documentation.